sample.env

```diff
@@ -1,13 +1,14 @@
 DOCKER_HOST_IP=
 #SDW_GROUP_ID=usdot.jpo.sdw
```

Comment:
Should the SDW_GROUP_ID be removed from here? It looks like we're not passing it in as an environment variable in any of the docker compose files

Reply:
You're right it doesn't seem to be getting used. This has been removed from the `sample.env` and `README.md` files.